ops/mainnet/prod/backend/main.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   backend "s3" {
-    bucket = "nxtp-terraform-mainnet-prod-backend"
+    bucket = "terraform-mainnet-prod-backend"
```

Comment:
This should not change, it result in inconsistent states between what there's deployed and what needs to be redeployed.

One option is migrating state, but that should be done manually and carefully

Reply:
Will work on migrating these manually.